run-devel.sh

```diff
@@ -31,20 +31,21 @@ while [ -n "$1" ]; do
     shift
 done
 
-CONTAINER="ros-$ROS_DISTRO-devel"
-IMAGE="ros-$ROS_DISTRO:devel"
+CONTAINER="orise-robotics_ros-$ROS_DISTRO-devel"
```

Comment:
I think the container name became very big. I think we could keep the way it was

Reply:
Maybe if someone has multiple containers just ros-distro-devel can generate some conflict, how about orise-ros-distro-devel?

Reply:
I prefer @ivan1993br's suggestion

Reply:
what about without ros prefix? like

`orise-<distro>-devel`

Reply:
sounds good too